scripts/build.py

```diff
@@ -93,6 +90,10 @@
 FILEPATHS_PROVIDED_TO_FRONTEND = (
     'images/*', 'videos/*', 'i18n/*', '*_directive.html', '*.png', '*.json')
 
+# When deleting files from staging directory, js suffix is replaced with ts
+# since files in source dir are ts files and hashes are mapped to them. The
```

Comment:
Hashes should be calculated based on JS files, not TS.

Reply:
Done.